### PR TITLE
[Feature] column cards

### PR DESCRIPTION
**Pull request by jonycfu**
**PR checklist**

- [ X] Ensure you have added or ran the appropriate tests for your PR.
- [ X] DCO signed

**What type of PR is this?**
UI Change 
**What this PR does / why we need it**:
Column Metrics v1
**Which issue(s) this PR fixes**:
27427
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
yes

New UI update for the dataset table column cards to display charts and specific column metrics

### CODE REVIEW

**Comment by jonycfu** [comment]
Note: `schema.json` updated with meta `description` for display on UI Column (i) info hover tooltip

**Comment by jonycfu** [comment]
Another note: 
code wise, `histogram` now replaces `distribution` (deprecated)

**Review by popcornylu — APPROVED** [review]
- [x] Average description icon: The mean average -> The mean average.  Mean is redundent?
- [x] Min/Max description icon: "Can be numeric or datetime string."  I think it is a description for developer, not for users. The users know it is datetime or numeric.
- [ ] integer icon. Show `123` rather than `?`
- [ ] The description of most common say it is mode. But we show the top 3 category. Is the description too confusing?


All these are good to have. So feel free to change or not. Others LGTM.

File: static_report/src/components/shared/ColumnCard/ColumnCardHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+function getIconForColumnType(columnDatum: ColumnSchema): {
+  backgroundColor: ColorProps['color'];
+  icon: any; //IconType not provided
+} {
+  const { type } = columnDatum;
+  const isCategorical = checkColumnCategorical(columnDatum);
+
+  if (isCategorical)
```
Comment (neighborhood999) [inline static_report/src/components/shared/ColumnCard/ColumnCardHeader.tsx:57]: I feel including brackets in `if` statement is more readable.

Maybe you can use the `switch` statement here.

Comment (jonycfu) [reply static_report/src/components/shared/ColumnCard/ColumnCardHeader.tsx:57]: I'm not a big fan of case/switch for the reason that conditional expressions are needed, and make it way harder to read. e.g. `switch(true) { ... }` 🤮 

See: https://stackoverflow.com/questions/2696436/switch-case-with-conditions
 
opting for explicit `{ .. }` instead

**Review by neighborhood999 — APPROVED** [review]

**Pull request merged**